app/controllers/spotlight/resources_controller.rb

```diff
@@ -44,7 +44,7 @@ def reindex_all
 
     protected
     def resource_params
-      params.require(:resource).permit(:url)
+      params.require(:resource).permit(:url, data: params[:resource][:data].try(:keys))
```

Comment:
@jkeck what do you think of this approach (from http://edgeguides.rubyonrails.org/action_controller_overview.html#outside-the-scope-of-strong-parameters)?